docs/roadmap.md

```diff
@@ -1,16 +1,16 @@
-## roadmap
+## RoadMap
 ### v0.1.0
-1. Standalone version, local storage
-2. Analytical storage format
-3. Support SQL
+- [x] Standalone version, local storage
+- [x] Analytical storage format
+- [x] Support SQL
 
 ### v0.2.0
-1. Distributed version supports static topology
-2. The underlying storage supports Alibaba Cloud oss
-3. Distributed WAL supports OBKV
-4. Support Prometheus protocol
+- [ ] Distributed version supports static topology
```

Comment:
Current version has supported the static topology so just mark it done.

Reply:
👍